### DISCUSSION TITLE
Bug in windows versions of GNS3 3.0 a1

### DISCUSSION THREAD

**Discussion by aksplease** [GNS3/gns3-gui#3370] [General]
hi Develop team ,

please address the issues faced in new gns3 3.0

not able to add a VMware instance ,might be this happening as the gns3 app not able to detect VMware location automatically. as during starting the app it was not automatically starting the VMware.

when gns3 app is started it not firing the gns3 vmware and gns3vm to start as its doing in  earlier verson(might be location maping issue )

readding the IOS for 7200 router did not gave a error but create error for 3745 series 

after uploading IOS it not sensing IOS presence one closing and staring again the uploading page sensing the IOS 

router is not being active on start ..

please go through the the error snapshots and find out errors with care 

it will be nice to make a small operational manual  about  the new features 
and how to work with new version .as there is many changes in new versions ... 

![IOS running issue](https://user-images.githubusercontent.com/11521667/183597345-5bc19db0-9f15-4be1-9697-1ede146f2a96.jpg)
![IOS UPLOAD ERROR](https://user-images.githubusercontent.com/11521667/183597358-55f791e3-2f58-442b-8c5d-9134828fb36f.jpg)
![IOS UPLOAD ERROR2](https://user-images.githubusercontent.com/11521667/183597363-720f3f26-cdd5-48a4-8aa8-ea674992fd17.jpg)
![IOS UPLOAD ERROR3](https://user-images.githubusercontent.com/11521667/183597366-21502e48-7119-4c32-b77d-54a049f220fa.jpg)
![VM instance error1](https://user-images.githubusercontent.com/11521667/183597372-929e9e9b-b407-461f-b08f-550d3a6b86fd.jpg)

**Comment by eantowne** [comment]
Vmware instances will not work unless you are running on Linux. The main server is the GNS3 VM, and VMware is not installed on it. The solution is to create custom QEMU images instead that are running inside GNS3.

Automatic GNS3 VM start/stop is no longer an option. There is no longer a local server, except for Linux hosts. It is required to manually start the GNS3 VM prior to GNS3, and manually stop after closing GNS3.

Documentation will most likely be released in the future, but not for this round of testing.

It would be exceedingly helpful if you were to create separate issue submission for each issue/bug you encounter. This enables the dev team to manage the information and solution of each issue easier. It also prevents confusion.

**Comment by aksplease** [comment]
thank you for your replay 

i couldn't understand your provided solutions 

The main server is the GNS3 VM, and VMware is not installed on it. The solution is to create custom QEMU images instead that are running inside GNS3.

i am running Gns3 Vm inside VMware workstation (16.2.3) in windows-11 OS and gns3 and Gns3Vm(inside VMware) running in same Machine.

1. when i run gns3 app for the first time it automatically did not connect to gns3Vm so in controller i had to change the gns3Vm server IP and login as Admin/admin as gns3/gns3 was not working 
2. after changing all the connection established between gns3app and gns3vm 
3. but as i uploaded cicso IOS router images for 7200/3745 it uploaded but it was not automatically making it green and when i clicking on finish it was giving error no image found 
4. after closing and reopening the template to add device when i select 7200 it sensed ios and added successfully.
5. when i add device to blank project and run it the device did not start .

i have shared all details snapshots ..

hopefully you understand the problems now .pls suggest the solution and make gns3 bit user friendly ..

**Reply by smart2128** [reply]
3.0.0 alpha 1 is tagged as a pre-release and it's the first development release in alpha stage, released for bug hunting and reporting, not for production purposes.
If you need a full working GNS3 installation, I would suggest you to install the latest stable release (2.2.33.1).

**Reply by aksplease** [reply]
thank you ,smart2128 for your replay ,


my intention was to highlight the bug area and draw your attention to the above points, hopefully you guys will keep above point in mind during fixing the bugs …

regards
